content/python/concepts/strings/terms/strip/strip.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: '.strip()'
+Description: 'Eliminates spaces at the beginning and end of a string, and specified characters when passed as a parameter.'
+Subjects:
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.strip()`** method removes any whitespace at the beginning and end of a string. It’s also possible to specify which characters will be removed; otherwise, all whitespaces will be removed. 
+
+## Syntax
+
+```pseudo
+string.strip(characters) 
+```
+
+Parameters are optional for this method.
+
+## Example 1
+
+The following example shows the usage of `.strip()` _without_ a parameter:
+
+```py
+string_with_white_spaces = "   221B, Baker Street   "
+
+print(string_with_white_spaces) 
+print(string_with_white_spaces.strip())
+```
+
+Output:
+
+```shell
+   221B, Baker Street   
+221B, Baker Street
+```
+
+In the first print statement, the string includes leading and trailing spaces.
+In the second print statement, `.strip()` removes these spaces, resulting in a clean output.
```

Comment:
```suggestion
```

Reply:
For most simple entry examples we don't include additional text explaining the output. Additional explanations are generally reserved for concept entries that include more material, or code blocks that may contain complex or unintuitive code.